gryzzly/Api/GryzzlyApiService.php

```diff
@@ -228,6 +233,7 @@ private function buildPaginationParameters(array $parameters): array
         return [
             'limit' => $perPage,
             'offset' => $offset,
+            ...$parameters,
         ];
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** il faut placer `...$parameters,` avant `limit` et `offset` car si `$parameters` contient les clés `limit` et `offset` ça ecrasera les valeurs qu'on définit juste au dessus‏

```suggestion
        return [
            ...$parameters,
            'limit' => $perPage,
            'offset' => $offset,
        ];
```